src/tools/gophish_export.py

```diff
@@ -400,6 +401,13 @@ def main() -> None:
             logging.critical(e.args[0])
             sys.exit(1)
 
+    if not validate_assessment_id(args["ASSESSMENT_ID"]):
+        logging.critical(
+            '"%s" is an invalid assessment_id format. Example: RV1234',
+            args["ASSESSMENT_ID"],
```

Comment:
This error message could be a bit more explanatory about what the valid format is, rather than just giving the example `RV1234`.

Reply:
Error message was modified to explain the assessment_id format.

See c6d29c5 for revisions